eth2near/contract_wrapper/src/near_contract_wrapper.rs

```diff
@@ -51,8 +51,8 @@ impl NearContractWrapper {
         let v: Value = serde_json::from_str(
             &std::fs::read_to_string(path_to_signer_secret_key).expect("Unable to read file"),
         )
-        .unwrap();
-        let signer_secret_key = trim_quotes(serde_json::to_string(&v["private_key"]).unwrap());
+        .expect("Error on parsing file with secret key during contract initialization");
+        let signer_secret_key = trim_quotes(serde_json::to_string(&v["private_key"]).expect("Error during trim quotes of signature secret key"));
```

Comment:
`expect` will exactly panic, but just with a custom error msg provided with it.

Reply:
Here it is Okey, because it is during initialization(one time at the start). So, in that case, we want the program to finish.